components/page-header/style/index.less

```diff
@@ -0,0 +1,90 @@
+@import '../../style/themes/default';
+@import '../../style/mixins/index';
+
+@pageheader-prefix-cls: ~'@{ant-prefix}-page-header';
+
+.@{pageheader-prefix-cls} {
+  .reset-component;
+  padding: 16px 32px;
+  position: relative;
+  background: @component-background;
+
+  &.@{pageheader-prefix-cls}-have-footer {
+    padding: 20px 32px;
+    padding-bottom: 0;
+  }
+
+  .@{pageheader-prefix-cls}-back-icon {
+    cursor: pointer;
+    font-size: 16px;
+    line-height: 16px;
```

Comment:
Line-height should always be unitless. I've noticed that throughout Ant, these are typically containing a unit based value, which cause all sorts of problems to children within elements that have parents' line-heights set to a unit based value.